development/compiling/compiling_for_osx.rst

```diff
@@ -32,7 +32,7 @@ Compiling
 
 Start a terminal, go to the root directory of the engine source code and type::
 
-    scons platform=osx --jobs=$(sysctl -n hw.logicalcpu)
+    python3 $(which scons platform=osx --jobs=$(sysctl -n hw.logicalcpu))
```

Comment:
Should actually be `python3 $(which scons) ...`.

But IMO it's better to leave it as is and add a `.. note::` about how to enforce running it with Python 3, or directly in the requirements, as done in https://docs.godotengine.org/en/latest/development/compiling/compiling_for_linuxbsd.html#requirements